docs/index.rst

```diff
@@ -1487,11 +1505,14 @@ Setup |ccloud|
 
       source delta_configs/env.delta
 
+.. _cp-demo-telemetry-reporter:
 
 Telemetry Reporter
 ------------------
 
-#. Create a new ``Cloud`` API key and secret to authenticate with |ccloud|. These credentials will be used by the :ref:`telemetry_reporter` and used by the Metrics API, which can be used for hosted on-prem clusters as well as |ccloud| clusters.
+Enable :ref:`telemetry_reporter` on the on-prem cluster, and configure it to send metrics to the |ccloud| instance created above..
```

Comment:
```suggestion
Enable :ref:`telemetry_reporter` on the on-prem cluster and configure it to send metrics to the |ccloud| instance created above..
```

Reply:
Since this isn't part of our standard public docs, this may not apply. However, our style is to use `on premises` instead of `on-prem` or `on prem` in docs. See https://confluentinc.atlassian.net/wiki/spaces/DOC/pages/161743785/Style+Guide+for+Confluent+Documentation. if you decide to use the approved style, do a search and replace (there are a lot of locations in the doc).

Reply:
@tobydrake7 the source is different but it all lands in docs.confluent.io, so if docs team prefers `on premises`, I can switch.  Please clarify if this guidance is merely suggested (in which case I will leave as-is) or strongly preferred (in which case I will update to `on premises`).

Reply:
> @tobydrake7 the source is different but it all lands in docs.confluent.io, so if docs team prefers `on premises`, I can switch. Please clarify if this guidance is merely suggested (in which case I will leave as-is) or strongly preferred (in which case I will update to `on premises`).

The doc team strongly prefers consistency and following the style guide. However, you can follow your heart on this. If you'd rather remain somewhat informal with how everyone says it (`on-prem`), in this tutorial type of document, go ahead. You know your audience for these things. If they look at `on premises` and think you're trying to use big words or something, use on-prem. 👍

Reply:
@tobydrake7 got it.  Lots of this is subjective but I'm in the `on-prem` camp, so I will leave it as is.  Thank you.